### DISCUSSION TITLE
auto/addons links are not refreshed/created after `invoke git-aggregate`

### DISCUSSION THREAD

**Discussion by patovala** [Tecnativa/doodba#536] [Doodba - The image] [answered]
Seems to be related to: [467](https://github.com/Tecnativa/doodba/discussions/467) but for me is happening in 15.0 branch. The addons.yml definitions are ignored and there are no links created for the new git-aggregated repos.

Not sure if a `ln -s path/to/custom/src/missing/module` inside the auto/addons is enough or what should be the way to solve it?

**Comment by pedrobaeza** [comment, marked as answer]
That links are created on instance initialization, not when running `invoke git-aggregate`. Don't base your development on such folder, but on each repository folder inside `src`, or always do `docker-compose up -d` / `docker-compose restart odoo` after aggregating.
